pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/impl/TopicTransactionBuffer.java

```diff
@@ -89,7 +91,11 @@ public class TopicTransactionBuffer extends TopicTransactionBufferState implemen
 
     private final int takeSnapshotIntervalTime;
 
+    @Getter
```

Comment:
Is this `@Getter` is only for the test?

Reply:
Thanks for you reminder. I mark it as `VisibleForTesting`.